libraries/AP_Avoidance/AP_Avoidance.cpp

```diff
@@ -507,10 +514,28 @@ void AP_Avoidance::handle_avoidance_local(AP_Avoidance::Obstacle *threat)
     MAV_COLLISION_THREAT_LEVEL new_threat_level = MAV_COLLISION_THREAT_LEVEL_NONE;
     MAV_COLLISION_ACTION action = MAV_COLLISION_ACTION_NONE;
 
+    // Get vehicle location for altitude
+    
+    Location my_loc;
```

Comment:
move this down to line just above to where it's used so no one accidently uses it by assuming it's valid

Reply:
here's an example: 
https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Landing/AP_Landing_Deepstall.cpp#L269